project/TestGroups.scala

```diff
@@ -34,7 +34,8 @@ object TestGroups {
       "tests.CompressionSuite", "tests.OpenBrowserSuite",
       "tests.FilenameLength", "tests.PrettyPrintTreeSuite",
       "tests.decorations.SyntheticDecorationsLspSuite",
-      "tests.codeactions.BracesParensLspSuite"),
+      "tests.codeactions.BracesParensLspSuite",
+      "tests.debug.CompletionDapSuite", "tests.debug.EvaluationDapSuite"),
```

Comment:
I haven't been aware that sth like this exists. What happens to test cases which don't belong to one of the sets?

Reply:
They get invoked in the first shard. I added a warning about actually.